source/accessibility.html.md.erb

```diff
@@ -0,0 +1,120 @@
+
+## Compliance status
+
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to the non-compliances listed below.
+
+### Non-accessible content
+
+The content listed below is non-accessible for the following reasons.
+
+#### Non-compliance with the accessibility regulations
+
+The following pages have redundant links:
+
+- [API reference](https://docs.payments.service.gov.uk/api_reference/#api-reference)
+- [Take a Direct Debit payment](https://docs.payments.service.gov.uk/direct_debit/#take-a-direct-debit-payment)
+- [Integrate with the GOV.UK API](https://docs.payments.service.gov.uk/integrate_with_govuk_pay/#integrate-with-the-gov-uk-pay-api)
+- [Take a payment over the phone (‘MOTO’ payments)](https://docs.payments.service.gov.uk/moto_payments/#take-a-payment-over-the-phone-moto-payments)
+- [Add corporate card fees](https://docs.payments.service.gov.uk/corporate_card_surcharges/#add-corporate-card-fees)
+- [Block prepaid cards](https://docs.payments.service.gov.uk/block_prepaid_cards/#block-prepaid-cards)
+- [Go live](https://docs.payments.service.gov.uk/switching_to_live/#go-live)
+- [Connect your live account to ePDQ](https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq)
+- [Connect your live account to Smartpay](https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay)
+- [Support](https://docs.payments.service.gov.uk/support_contact_and_more_information/#support)
+
+This is not a fail, but should be looked at under [WCAG 2.1 success criterion 2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html).
+
+Also, some parts of this website are not fully accessible because of [issues caused by our Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/accessibility#using-the-technical-documentation-template-for-your-own-documentation).
+
+We plan to fix the accessibility issues with the Technical Documentation Template by [[date]].
+
+## How we tested this website
+
+We last tested this website for accessibility issues in May 2020.
+
+We used manual and automated tests to look for issues such as:
+
+- lack of keyboard accessibility
+- link text that’s not descriptive
+- non-unique or non-hierarchical headings
+- italics, bold or block capital formatting
+- inaccessible formatting in general
+- inaccessible language
+- inaccessible diagrams or images
+- lack of colour contrast for text, important graphics and controls
+- images not having meaningful alt text
+- problems when using assistive technologies such as screen readers and screen magnifiers
+
+## What we’re doing to improve accessibility
+
+We plan to look at the redundant links by [date].
```

Comment:
do we need to commit to a date?

Reply:
We do. I've checked with the accessibility community and there is not current guidance or advice on when this is. Maybe we could say end of 2020 or maybe end of Q1 2020?